application-ownership-models/README.md

```diff
@@ -0,0 +1,116 @@
+# Application ownership models
+
+A lifecycle of an Kubernetes application could be roughly split into the following stages and sub-stages:
+- Development
+- Release engineering, which could be broken down to:
+  - Building
+  - Testing
+  - Packaging
+  - Delivery (deployment, upgrades, removal)
+- Operations, which could be broken down to:
+  - Investigating issues
+  - Improving performance
+  - On-call
+  - Postmortems
+  - Root cause analysis
+
+Additionally, while for the most of applications it's obvious which team can and should take care of its lifecycle, for some apps that is not very clear, as multiple teams have equally important stakes in it.
+
+Based on the above, we can define two different criteria for defining application ownership models:
+- What stages of the application lifecycle are owned by a team?
+- Is there one or more Giant Swarm teams that can be considered owners of the application?
+
+## 1. Application lifecycle ownership
+
+We can define two high-level application ownership models:
+- Full application ownership
+- Partial application ownership, usually including release engineering (full or partial) and operations. This is typical for upstream applications where the development is done outside of Giant Swarm.
```

Comment:
But GS could collaborate on that part  (as we are doing now in CAPVCD)

Reply:
Yeah, definitely, we can and we should work on those, but it's still an outside/upstream project which is under another organization. Thanks for the comment, I'll clarify it a bit more.

Reply:
I think this should not be part of the RFC. Upstream contributions still needs to be approved by them, so we can only offer changes. I would not call this real development and omit from the RFC for keeping things simple.